internal/controller/fluxinstance_controller.go

```diff
@@ -516,6 +525,20 @@ func (r *FluxInstanceReconciler) apply(ctx context.Context,
 		}
 	}
 
+	// Send event to notification-controller only if the server-side apply resulted in changes.
+	applyLog := strings.TrimSuffix(changeSetLog.String(), "\n")
+	if applyLog != "" {
+		action := "updated"
+		if len(oldInventory.Entries) == 0 {
+			action = "installed"
+		}
+
+		msg := fmt.Sprintf("Flux %s revision %s\n%s", action, buildResult.Revision, applyLog)
```

Comment:
```suggestion
		msg := fmt.Sprintf("Flux revision %s %s\n%s", buildResult.Revision, action, applyLog)
```

I think `Flux revision X installed` reads better than `Flux installed revision X` since Flux is the one being installed, and not the one installing anything. Same for `updated`.

Reply:
How about using the verb first, like `Installed Flux revision X`, I want to place the revision last as it contains a SHA256 which is so long.

Reply:
Sounds good 👍

Reply:
I went with "Flux <version> [installed|updated]" and moved the revision to the event metadata.